sympy/functions/special/error_functions.py

```diff
@@ -1942,7 +1942,6 @@ def _eval_is_zero(self):
         if z.is_zero:
             return True
 
-
```

Comment:
Deletion of this line is unnecessary.

Reply:
sir but it is not affecting the code and it was un necessary line